providers/common/capabilities.c

```diff
@@ -204,6 +210,8 @@ static const OSSL_PARAM param_group_list[][10] = {
     TLS_GROUP_ENTRY("ffdhe6144", "ffdhe6144", "DH", 36),
     TLS_GROUP_ENTRY("ffdhe8192", "ffdhe8192", "DH", 37),
 # endif
+    /* TODO(ML-KEM): Decide final name, e.g., ML-KEM768 or MLKEM768 */
+    TLS_GROUP_ENTRY("MLKEM-768", "MLKEM-768", "MLKEM-768", 38),
```

Comment:
IANA calls the TLS supported group "MLKEM768". NIST calls the algorithm "ML-KEM-768". So I would use those conventions when referring to the TLS group vs the algorithm, i.e.

```` c
TLS_GROUP_ENTRY("MLKEM768", "MLKEM768", "ML-KEM-768", 38),
````

Reply:
Hmm. The alg name now matches - but the group name still differs from the IANA name.

Reply:
Hmm -- maybe I forgot to add a file to the previous commit. Should be fixed with https://github.com/openssl/openssl/pull/25848/commits/7b7cdfb693855255e65ff6968c491f55688b0996